homeassistant/components/camera/netatmo.py

```diff
@@ -76,7 +76,7 @@ def camera_image(self):
                     self._localurl), timeout=10)
             else:
                 response = requests.get('{0}/live/snapshot_720.jpg'.format(
-                    self._vpnurl), timeout=10)
+                    self._vpnurl), timeout=10, verify=False)
```

Comment:
Ok thanks for your clarification @balloob, but as Netatmo fixed their ping mechanism (used to get the camera's local url) this change is less important, I suggest to remove this change as it is needed only for person who have their camera on a different network than HASS, so it can wait for a major update. (The rest of fix for the netatmo tags -binary sensor- is still need).

I'm not a certificate expert but I think that Raspbian Jessie does not have the right root certificate as Netatmo don't use self signed certificate, and this url was working fine on my dev setup (MacOSX)